src/components/Messages/CreateMessageModal/MessagesModalList.tsx

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import { List } from "@mui/material";
+import { useAppSelector } from "../../../state/hooks";
+import MessagesModalListItem from "./MessagesModalListItem";
+
+export type OtherUser = {
```

Comment:
I think I said I'd leave it up to you to decide what to do with this type, did you decide to keep it? Just curious

Reply:
Yeah I decided to keep it. I felt like it has a different purpose than `SelectedConversation`.

One thing we could do is rename it to something general so that it's used in both components and then apply that type individually for the required state. 

I might do that rather than have duplicate types

Reply:
Renamed `SelectedConversation` to `SelectedUser` to fit the criteria for both `OtherUser` and `SelectedCoversation`.